README.md

```diff
@@ -0,0 +1,51 @@
+# Wikispeech-Prerender
+
+Starting from scratch will 
+
+## REST
+
+Most REST calls are exist for debug and development reasons.
+As a user of this service, all you need is ```POST /api/wiki```
+
+### POST /api/wiki
```

Comment:
Would be good with a one-liner describing what the call does (similar to what you added to many of the calls below)